nservicebus/upgrades/5to6.md

```diff
@@ -301,12 +301,14 @@ snippet:5to6ScanningUpgrade
 
 ## Throttling
 
-Throttling of message processing can no longer be adjusted at run time and will as of Version 6 require the endpoint to be restarted to take effect. You can read more about throttling [here](/nservicebus/operations/tuning.md).
+Requirements to throttling mechanisms are very different. While some 3rd party services (e.g. GitHub, Twitter, Google, etc.) enforce rate limits on certain time periods, other services may have entirely different usage limitations. The previous throttling API offered a very limited, messages per second based, throttling mechanism which only works for very few scenarios. Therefore the throttling API has been removed with Version 6 without a built-in alternative. [Tuning NServiceBus](/nservicebus/operations/tuning.md) contains more information about implementing a custom throttling mechanism.
+
+The `MaximumMessageThroughputPerSecond` on the `TransportConfig` class has been marked as obsolete. Using a configuration based approach, the endpoint will fail to start when using the `MaximumMessageThroughputPerSecond` attribute on the `<TransportConfig>` element.
 
 
 ## RepeatedFailuresOverTimeCircuitBreaker has been made internal
 
-If your are using it just include [the source code](https://github.com/Particular/NServiceBus/blob/5.2.5/src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs) in your project.
+If you are using it just include [the source code](https://github.com/Particular/NServiceBus/blob/5.2.5/src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs) in your project.
```

Comment:
@SimonCropp that chapter is actually not from my changes. I just fixed the "your are", but I'm happy to fix it. How about:
*We recommend to use a independent Circuit Breaker implementation. Please feel free to copy [this implementation].*